.github/workflows/render-svg.yaml

```diff
@@ -0,0 +1,47 @@
+name: Render SVG files
+
+on:
+  push:
+    paths:
+      - man/**logo.svg
+
+jobs:
+  render-svg:
+    runs-on: windows-latest
+
+    steps:
+      - uses: actions/checkout@v3
+
+      - uses: r-lib/actions/setup-r@v2
+
+      - name: Install dependencies
+        uses: r-lib/actions/setup-r-dependencies@v2
+        with:
+          packages: |
+            any::magick
+            any::rsvg
+
+      - name: Render logo SVG
+        run: |
+          library(magick)
+
+          svg_path <- list.files("man/figures", "*logo.svg", full.names = TRUE)
+
+          stopifnot(length(svg_path) == 1)
+
+          image_read_svg(svg_path) |>
+            image_crop("100%x50%") |>
+            image_trim() |>
+            image_resize("x250") |>
+            image_write("man/figures/logo.png")
```

Comment:
Before, the logo was rendered at a high resolution and then downscaled with `usethis::use_logo()`
This function outputs the logo at a resolution of 240x278 or, it would seem, as close to as possible.
I don't know if it gives some unforeseen issues down the line that we here get images that are 217x250 ?

Reply:
Good point. I've changed the size to be closer to the previous­ size. Closest I can get is 240x277, but hopefully this will not break anything.